images/minio/main.tf

```diff
@@ -8,47 +8,59 @@ variable "target_repository" {
   description = "The docker repo into which the image and attestations should be published."
 }
 
+locals {
+  packages = {
+    "minio" : "minio",
+    "minio-client" : "mc",
+  }
+}
+
 module "latest" {
-  source = "../../tflib/publisher"
+  for_each = local.packages
+  source   = "../../tflib/publisher"
 
   name = basename(path.module)
 
-  target_repository = var.target_repository
-  config            = file("${path.module}/configs/latest.apko.yaml")
+  target_repository = "${var.target_repository}${trimprefix(each.key, "minio")}"
+  config            = file("${path.module}/configs/latest.${each.key}.apko.yaml")
 }
 
 module "dev" { source = "../../tflib/dev-subvariant" }
 
 module "latest-dev" {
-  source = "../../tflib/publisher"
+  for_each = local.packages
+  source   = "../../tflib/publisher"
 
   name = basename(path.module)
 
-  target_repository = var.target_repository
+  target_repository = "${var.target_repository}${trimprefix(each.key, "minio")}"
   # Make the dev variant an explicit extension of the
   # locked original.
-  config         = jsonencode(module.latest.config)
+  config         = jsonencode(module.latest[each.key].config)
   extra_packages = module.dev.extra_packages
 }
 
 module "version-tags" {
-  source  = "../../tflib/version-tags"
-  package = "minio"
-  config  = module.latest.config
+  for_each = local.packages
+  source   = "../../tflib/version-tags"
+  package  = each.value
+  config   = module.latest[each.key].config
 }
 
 module "test-latest" {
-  source = "./tests"
-  digest = module.latest.image_ref
+  for_each = local.packages
+  source   = "./tests"
+  digest   = module.latest[each.key].image_ref
```

Comment:
We should make this pass all of the constituent images as `digests = { ... }` and then we can run the current `--version` test on each, which will make the follow up to test them against each other much more contained.

Reply:
Done